stdlib/ctypes/__init__.pyi

```diff
@@ -148,7 +149,7 @@ c_buffer = create_string_buffer
 
 def create_unicode_buffer(init: int | str, size: int | None = None) -> Array[c_wchar]: ...
 @deprecated("Deprecated in Python 3.13; removal scheduled for Python 3.15")
-def SetPointerType(pointer: type[_Pointer[Any]], cls: Any) -> None: ...  # noqa: F811
+def SetPointerType(pointer: type[_Pointer[Any]], cls: Any) -> None: ...
```

Comment:
While we're here: Can we either comment the `Any` or just remove the type annotation for now?

Reply:
It can really be `Any`, what comment would you prefer for this case?

Reply:
Just to clarify: I mean the `cls: Any`, not the `_Pointer[Any]`. (I think we need slight revisions to the "comment all `Any`" policy.) What does the `Any` mean here, what are the accepted types? It says `cls`, so should it be really `type`? And in that case, is every type accepted? Or is literally everything accepted, in which case it should be `object`?

I tried to find this out myself to make a suggestion, but it turns out `SetPointerType` is undocumented (and deprecated, of course), so I had no luck.